packages/nimble-blazor/NimbleBlazor/Components/NimbleRadioGroup.razor.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Components;
+
+namespace NimbleBlazor;
+
+public partial class NimbleRadioGroup : NimbleInputBase<string>
+{
+    [Parameter]
+    public bool? Disabled { get; set; }
```

Comment:
I think we're now trying to get in the habit of documenting the public APIs.

Reply:
Added docs